src/script/api/script_client.cpp

```diff
@@ -0,0 +1,53 @@
+/* $Id$ */
+
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file script_client.cpp Implementation of ScriptClient. */
+
+#include "../../stdafx.h"
+#include "script_client.hpp"
+#include "../../network/network.h"
+#include "../../network/network_base.h"
+
```

Comment:
Does this module compile with "./configure --disable-network"?

Reply:
should probably be checked by CI btw